doc/internal/man3/OSSL_METHOD_STORE.pod

```diff
@@ -95,6 +97,10 @@ The result, if any, is returned in I<method>.
 ossl_method_store_cache_set() sets a cache entry identified by I<nid> with the
 property query I<prop_query> in the I<store>.
 Future calls to ossl_method_store_cache_get() will return the specified I<method>.
+If the I<method_up_ref> function is given, it's called to increment the
+reference count of the method.
+If the I<method_destruct> function is given, it's called when this
+implementation is being freed from the I<cache>.
```

Comment:
This needs to be rephrase, since they are mandatory

Reply:
Agreed, will change.

Reply:
Done.